### PR TITLE
Optimize

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -97,7 +100,7 @@ <h1 class="page_banner__title">{{ page.title }}</h1>
     </div>
 </footer>
 
-<script src="/js/jquery.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
Comment (dragoonis) [inline _layouts/default.html]: Change this to src="//ajax...."

**Comment by dragoonis** [comment]
@reinink can you comment on these changes?

File: _layouts/default.html (hunk 1 above)
Comment (PeterDaveHello) [inline _layouts/default.html]: https is safer, now google cdn, cdnjs and jsdelivr provides https prefix url by default, what do you think?

Comment (PeterDaveHello) [inline _layouts/default.html]: BTW, Google use http/2.0 or QUIC, which uses https, but be very fast

**Comment by reinink** [comment]
The image optimizations look good, but I'm very new to the `rel="preconnect"` technique, but that's probably because it's a new feature just hitting modern browsers. I've pulled this down and run it locally, and everything looks fine to me.

**Comment by dragoonis** [comment]
Thanks @PeterDaveHello for your contribution.

**Comment by PeterDaveHello** [comment]
:+1:

**Pull request merged**